mobs/malaga.yml

```diff
@@ -13,13 +13,14 @@ schedule:
     duration: 120
 background_color: Peru
 text_color: SeaShell
-links:  
+links:  https://alterisian.github.io/helloworld
```

Comment:
This is the cause of the failure. Please see the `links` property:
https://mobusoperandi.com/add.html